examples/sampling/node_classification.py

```diff
@@ -0,0 +1,240 @@
+"""
```

Comment:
Add a sentence describing what this script does. For example, node classification with neighbor sampling.

Reply:
Get~